qstack/regression/regression.py

```diff
@@ -3,16 +3,16 @@
 import numpy as np
 import scipy
 from sklearn.model_selection import train_test_split
+from sklearn.metrics import r2_score
```

Comment:
**maybe not needed**
_make separate utility to computes various metrics (**new issue ?**) ?_

Reply:
Actually, it might also be relevant for the project with Puck and the Reiher group.
_I'll make an issue/pr draft._